ceph_deploy/util/help_formatters.py

```diff
@@ -0,0 +1,11 @@
+import argparse
+
+
+class ToggleRawTextHelpFormatter(argparse.HelpFormatter):
+    """Inspired by the SmartFormatter at
+       https://bitbucket.org/ruamel/std.argparse
+   """
+    def _split_lines(self, text, width):
+        if text.startswith('R|'):
```

Comment:
this looks like a big gotcha that I didn't understand until I got the end of the review.  Would you mind adding a more thorough docstring with some examples so it is easier to parse/understand how it works?